### ISSUE TITLE
[Hadoop AI] Cannot get GPU status correctly

### ISSUE THREAD

**Issue by abuccts** [microsoft/pai#2736]
Currently, Hadoop AI patch cannot get GPU status correctly in node manager, including GPU index/number, ECC code, decommission unavailable GPU.
```
19/04/29 08:01:21 ERROR util.SysInfoLinux: ignored error: gpu 29 ECC code is 2019, will make this gpu unavailable
19/04/29 08:02:23 ERROR util.SysInfoLinux: ignored error: gpu 29 ECC code is 2019, will make this gpu unavailable
19/04/29 08:02:23 ERROR util.SysInfoLinux: ignored error: gpu 0 ECC code is 56174, will make this gpu unavailable
19/04/29 08:02:23 ERROR util.SysInfoLinux: ignored error: gpu 1 ECC code is 56175, will make this gpu unavailable
19/04/29 08:02:23 ERROR util.SysInfoLinux: ignored error: gpu 2 ECC code is 56176, will make this gpu unavailable
19/04/29 08:02:23 ERROR util.SysInfoLinux: ignored error: gpu 3 ECC code is 56177, will make this gpu unavailable
19/04/29 08:02:23 ERROR util.SysInfoLinux: ignored error: gpu 4 ECC code is 56178, will make this gpu unavailable
19/04/29 08:02:23 ERROR util.SysInfoLinux: ignored error: gpu 5 ECC code is 56180, will make this gpu unavailable
19/04/29 08:02:23 ERROR util.SysInfoLinux: ignored error: gpu 6 ECC code is 56182, will make this gpu unavailable
19/04/29 08:02:23 ERROR util.SysInfoLinux: ignored error: gpu 7 ECC code is 56183, will make this gpu unavailable
19/04/29 08:02:23 ERROR util.SysInfoLinux: ignored error: gpu 8 ECC code is 56184, will make this gpu unavailable
19/04/29 08:02:23 ERROR util.SysInfoLinux: ignored error: gpu 9 ECC code is 56185, will make this gpu unavailable
19/04/29 08:02:23 ERROR util.SysInfoLinux: ignored error: gpu 10 ECC code is 56186, will make this gpu unavailable
19/04/29 08:02:23 ERROR util.SysInfoLinux: ignored error: gpu 11 ECC code is 56188, will make this gpu unavailable
19/04/29 08:02:23 ERROR util.SysInfoLinux: ignored error: gpu 12 ECC code is 56190, will make this gpu unavailable
19/04/29 08:02:23 ERROR util.SysInfoLinux: ignored error: gpu 13 ECC code is 56200, will make this gpu unavailable
19/04/29 08:02:23 ERROR util.SysInfoLinux: ignored error: gpu 14 ECC code is 56201, will make this gpu unavailable
19/04/29 08:02:23 ERROR util.SysInfoLinux: ignored error: gpu 15 ECC code is 56202, will make this gpu unavailable
```

**Comment by fanyangCS** [comment]
deferred to pure k8s version.

**Comment by mzmssg** [comment]
Fixed in #2681
